binder/src/main/java/io/grpc/binder/internal/BinderServer.java

```diff
@@ -165,6 +174,28 @@ public synchronized boolean handleTransaction(int code, Parcel parcel) {
     return false;
   }
 
+  static class GoAwayHandler implements LeakSafeOneWayBinder.TransactionHandler {
```

Comment:
Nit: make it `final` since it's still visible to other classes within this package.

Reply:
done